### ISSUE TITLE
eval_results do not contain the actual answer, right?

### ISSUE THREAD

**Issue by emanuelevivoli** [TIGER-AI-Lab/MMLU-Pro#17]
Hello 👋 and thanks for the ✨ Amazing work! 
I'm a big fan of your works both in the models and the benchmarks!

My question is quite simple: is the "answer" key in the files in `eval_results` the actual answer for the question? If so, I'm worried about having the actual answer for the benchmark publicly available... Isn't that going to be in the "next-foundation-model-training-set" by any chance?

Thanks for your time,
Lele

**Comment by wenhuchen** [comment]
Yes, that could be a concern. I totally agree.

**Comment by emanuelevivoli** [comment]
Well, at least we are on the same opinion 👍

A solution would be to have an open validation set (even on higgingface as you are doing) where people can see what the questions and answers look like, and then something like a held-out test set that you evaluate models on. And for this, sadly but necessary, not releasing the annotations but managing a "competition server" with a leaderboard. 

We are doing this for many document/reading competitions (also comics!) in [1] it takes time, but it's more reliable 😉
The problem with not doing this is that in a year (or months, with the speed research is done nowadays) your test set will be in some model's training set [2]. And that is very sad, considering the effort in cleaning data, creating the questions, making it difficult and manually checking almost everything (I know the pain).

Thank you for your consideration,
Emanuele 

[1] [Robust reading competition](https://rrc.cvc.uab.es)
[2] https://x.com/polynoamial/status/1835086680266883205?t=7NDUfLn4BCzNwsyT6siw4g&s=19
